modules/user_data/templates/install_graphdb.sh.tpl

```diff
@@ -114,7 +114,28 @@ if [ "$graphdb_device: data" = "$(file -s $graphdb_device)" ]; then
   mkfs -t ext4 $graphdb_device
 fi
 
-mount $graphdb_device /var/opt/graphdb/data
+disk_mount_point="/var/opt/graphdb/data"
+
+# Check if the disk is already mounted
+if ! mount | grep -q "$graphdb_device"; then
```

Comment:
Why do we need this? This script runs only when the machine is first created and with ASG, when a machine is rebooted a new one will take its place.

Reply:
Well, if you reboot the machine from the AWS console, it just reboots, the ASG doesn't spin a new one

Reply:
you mean EC2 > Instances > Instance state > Reboot?

Reply:
yes

Reply:
I'm getting a warning message, saying that it "might" launch a new instance. I guess the catch is that it might :D